### PR TITLE
feat: create membership.md [DONT MERGE] [NEEDS DISCUSSION]

### PR DESCRIPTION
**Pull request by castrojo**
Initial cut of documented governance, this is long overdue and will likely require many revisions and tweaks. And as you can see, unfinished links all over the place.

### CODE REVIEW

File: docs/membership.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of one of the Universal Blue GitHub organizations comes with
+an elevated set of permissions. These capabilities should not be used by those
+that are not familiar with the current state of the Kubernetes project.
```
Comment (akdev1l) [inline docs/membership.md:128]: "Kubernetes project" maybe a typo?

Comment (castrojo) [reply docs/membership.md]: Got it!

File: docs/membership.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of one of the Universal Blue GitHub organizations comes with
+an elevated set of permissions. These capabilities should not be used by those
+that are not familiar with the current state of the project.
+
+Therefore members with an extended period away from the project with no activity
```
Comment (jberkus) [inline docs/membership.md:139]: probably want to set an actual period of time, that way you can script checking

File: docs/membership.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+Inactive members are defined as members of one of the Universal Blue Organizations
+with **no** contributions across any organization over the past 6 months. This is
+measured by GitHub activity.
+
+After an extended period away from the project with no activity
+those members would need to re-familiarize themselves with the current state
+before being able to contribute effectively.
+
+
+## Owner
```
Comment (jberkus) [inline docs/membership.md:155]: Wait, so is Owner a community status, or just a GH status?  This section doesn't make that clear.

File: docs/membership.md (hunk 2 above)
Comment (bigpod98) [reply docs/membership.md:139]: Next paragraph it says no activity in past 6 months 6 months

File: docs/membership.md (hunk 3 above)
Comment (castrojo) [reply docs/membership.md:154]: Made that more explicit, thanks for the review so far Josh, it means a lot!

**Comment by castrojo** [comment]
Alright, last call! I'll give it another day or so now that we've got the github permissions sorted. Thanks so much for the reviews folks, it means a bunch.

**Review by KyleGospo — APPROVED** [review]

**Review by bigpod98 — APPROVED** [review]

**Pull request merged**